source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,322 @@
+  <td> Payer has visited the `next_url` and landed on the GOV.UK Pay page to enter their account details. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> STARTED </code></td>
+  <td nowrap><code> PENDING </code></td>
+  <td> Payer has provided their account details , and confirms their consent to the agreement. <code>PENDING</code> is only used with delayed payment types such as Direct Debit. This indicates that payer consent has been given but the agreement has not yet been confirmed with the issuing bank. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> STARTED </code></td>
+  <td nowrap><code> CANCELLED - USER NOT ELIGIBLE </code></td>
+  <td> Payer has indicated that they are not the sole signatory for their bank account, and selects the link to go back and try a different payment method. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> PENDING </code></td>
+  <td nowrap><code> ACTIVE </code></td>
+  <td> The issuing bank has confirmed that funding source (for example, Direct Debit mandate) is valid. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> PENDING </code></td>
+  <td nowrap><code> FAILED </code></td>
+  <td> The issuing bank has rejected the funding source. Possible reasons: incorrect bank account details, bank account requires multiple signatories to set up a Direct Debit. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> PENDING </code></td>
+  <td nowrap><code> CANCELLED </code></td>
+  <td> Cancelled by the payer. </td>
+  </tr>
+  <tr>
+  <td nowrap><code> ACTIVE </code></td>
+  <td nowrap><code> CANCELLED </code></td>
+  <td> Cancelled by the payer. </td>
+  </tr>
+  </table>
+
+## One-off payments using Direct Debit
+
+To set up a Direct Debit mandate to take a single payment:
+
+* create an API key linked to your Direct Debit service
+
+* follow the instructions in the [“Payment flow” section](/payment_flow)
+
+The same API call is used for single payments in all contexts, including
+Direct Debit.
+
+You cannot take further payments from users using this method without creating
+a new payment. This requires users to re-enter their account details.
+
+## Variable payments using Direct Debit
+
+A variable agreement allows you to collect payments of any amount at any time.
+For example you may want to collect:
+
+* money each time a service is provided
+
+* payments where the amounts vary according to service usage
+
+* regular payments where you do not know in advance how much each payment will be
+
+You could also use the variable functionality to simulate a recurring or
+scheduled Direct Debit payment, by sending an API call on a set schedule.
+Users would be emailed each time a payment is collected against their mandate.
+
+### Variable Direct Debit payment process
+
+These steps happen in order.
+
+#### 1. Agreement is created
+
+API calL:  `POST /v1/agreements/`
+
+Your service must specify:
+
+* a `return_url` where GOV.UK Pay will direct the user when they have finished setting up the agreement
+
+* the `agreement_type` (`on demand`)
```

Comment:
is "on demand" an agreement type, or is there other agreement types?

Reply:
Ah yes interesting point. It's synonymous with "variable" but I'm not sure why in the context of the data here it's termed "on demand". @katiebates

Reply:
it's on-demand because that's what we were calling it at the time; @alexbishop1 can we refactor it to 'variable' without causing an issue?
@jonathanglassman there will (probably) be other agreement types in future - e.g. recurring/scheduled DD, so that's why we ask services to specify the type.